android/sample/src/main/java/com/facebook/spectrum/sample/model/ConfigurationViewModel.kt

```diff
@@ -75,8 +75,7 @@ enum class ConfigurationViewModel {
     )
 
     private val pngCompressionLevel: Array<EnumParameterEntry<Int>> = arrayOf(
-            EnumParameterEntry("Default", null, -1),
```

Comment:
`EnumParameterEntry("Default", null, null as Int?),`

and then in line 114:

```
EnumParameter("Compression level", pngCompressionLevel) { cb, v -> v?.let { cb.setCompressionLevel(it) } })
```

Reply:
Thanks :)
I'll change the code to fit your review point.

Reply:
I changed the code.
Please review it again :)